packages/e2e-tests/specs/editor/various/reusable-blocks.test.js

```diff
@@ -35,6 +37,7 @@ describe( 'Reusable blocks', () => {
 		await insertBlock( 'Paragraph' );
 		await page.keyboard.type( 'Hello there!' );
 
+		await showBlockToolbar();
```

Comment:
I don't think these showBlockToolbar calls are necessary. `clickBlockToolbarButton` already  calls it.

Reply:
Thanks for catching that!